### DISCUSSION TITLE
Allow include 3rd party files to HWS and Store Commerce extensions installers

### DISCUSSION THREAD

**Discussion by alsolonets** [microsoft/Dynamics365Commerce.Solutions#24] [Q&A] [answered]
Currently when creating HWS and Store Commerce extension installers, it is only possible to include .dll files to the extension installation package, for example, 3rd party drivers. However, some drivers require additional assets for their work. That was possible in the legacy Retail SDK, but not in the new model. Please add the ability to include any files to the extension installers.

**Comment by madyke** [comment, marked as answer]
Hello @alexeysolonets - thank you, we have opened an issue to track this as a feature request:
[Allow include 3rd party files to HWS and Store Commerce extensions installers](https://github.com/microsoft/Dynamics365Commerce.Solutions/issues/25)

**Reply by madyke** [reply]
This has been checked in, details can be found on the issues page:
[Allow include 3rd party files to HWS and Store Commerce extensions installers](https://github.com/microsoft/Dynamics365Commerce.Solutions/issues/25)
